src/resources/InProductExperiences/InProductExperiences.ts

```diff
@@ -0,0 +1,59 @@
+import API from '../../APICore';
+import Resource from '../Resource';
+import SearchPages from '../SearchPages/SearchPages';
+import {
+    InProductExperienceModel,
+    CreateInProductExperienceModel,
+    UpdateInProductExperienceModel,
+    InProductExperienceLoader,
+} from './InProductExperiencesInterfaces';
+
+/**
+ * In-Product Experiences (also known as In App Widgets) Swagger documentation can
+ * be found at https://platform.cloud.coveo.com/docs?urls.primaryName=Search%20Pages#/In-App,
+ * and shares certain functionality related to IPX interfaces with Search Pages. Swagger
+ * documentation for Search Pages can be found at
+ * https://platform.cloud.coveo.com/docs?urls.primaryName=Search%20Pages#/Search%20pages
+ */
+export default class InProductExperiences extends Resource {
+    static baseUrl = `/rest/organizations/${API.orgPlaceholder}`;
+    static ipxBaseUrl = `${InProductExperiences.baseUrl}/pages/inappwidget`;
+
+    /**
+     * IPX takes advantage of SearchPages for underlying the IPX interface management.
+     * This provides a helper around some methods, but for complete support for e.g. JS/CSS
+     * resource management, use this instance of `SearchPages` directly.
+     */
+    readonly searchPages: SearchPages;
+
+    constructor(api: API, serverlessApi: API) {
+        super(api, serverlessApi);
+        this.searchPages = new SearchPages(api, serverlessApi);
+    }
+
+    create(ipx: CreateInProductExperienceModel) {
+        return this.api.post<InProductExperienceModel>(InProductExperiences.ipxBaseUrl, ipx);
```

Comment:
Are you sure you got the response right for this call ? Ils been a while since I coded in the platform client but I think this might not be the right interface.

![2021-05-17_14-04](https://user-images.githubusercontent.com/45688129/118535883-04f28480-b719-11eb-82df-7e0d702841dd.png)

Reply:
Yeah these look right, the [`InProductExperienceModel'`](https://github.com/coveo/platform-client/pull/313/files/6a440a700ee5d1052fe01e447f3505ed45acabe3#diff-693ea1980cdd893a16364acb914706abb31645296ce65f21987ecd1ac3be6615R24) returned in that call matches that response type. The request and [response type](https://github.com/coveo/platform-client/pull/313/files/6a440a700ee5d1052fe01e447f3505ed45acabe3#diff-693ea1980cdd893a16364acb914706abb31645296ce65f21987ecd1ac3be6615R89)s vary heavily.

Reply:
My amend caused the referenced commit to disappear so those references don't work :fire: 

The screenshot you included matches the return type of the call, [`InProductExperienceModel`](https://github.com/coveo/platform-client/pull/313/files#diff-693ea1980cdd893a16364acb914706abb31645296ce65f21987ecd1ac3be6615R24)

Reply:
Update was just pushed to include a specific response type for the Create call :+1: